src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.commands;
+
+import java.util.List;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
+            + "Displays all persons in the address book as a list in sorted order.\n\t"
+            + "Example: " + COMMAND_WORD;
+
+
+    @Override
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.